.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: required
+
+services:
+ - docker
+
+before_install:
+ - docker pull dalg24/vera_tpls
+ - docker run -v $PWD:/mnt/Futility dalg24/vera_tpls /bin/sh -c "mkdir /build; cd /build; cmake /mnt/Futility; make; ctest"
+
+script:

```

Comment:
Note I HAD to define something in script.  Pull request #29 shows that it catches a failing build if the docker run throws an error code.